workflow-templates/release-branch.properties.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "adambkaplan Release Brancher",
```

Comment:
```suggestion
    "name": "Release Brancher",
```

Reply:
oops - forgot to fix that from my PoC copy/paste